miasm2/os_dep/win_api_x86_32.py

```diff
@@ -1963,6 +2044,72 @@ def user32_IsCharAlphaNumericA(jitter):
     jitter.func_ret_stdcall(ret_ad, ret)
 
 
+def get_fmt_args(jitter, fmt, cur_arg, get_str, char_size=1):
```

Comment:
Instead of copying `get_fmt_args` from `linux_stdlib`, it would be probably better to have them in common in `os_dep/common.py`; this way, they will both benefit from future update and fix...

Reply:
Right, this is an extract from a previous "work session" with miasm. I forgot that I copy-pasted it from `linux_stdlib`